--- config.h	2017-04-05 23:05:25.770972754 +0300
+++ config.h	2017-04-05 23:13:27.984327301 +0300
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Hack:pixelsize=12:antialias=true:autohint=true";
 static int borderpx = 2;

 /*
@@ -25,7 +25,7 @@

 /* Kerning / character bounding-box multipliers */
 static float cwscale = 1.0;
-static float chscale = 1.0;
+static float chscale = 1.1;

 /*
  * word delimiter string
@@ -42,7 +42,7 @@
 static int allowaltscreen = 1;

 /* frames per second st should at maximum draw to the screen */
-static unsigned int xfps = 120;
+static unsigned int xfps = 60;
 static unsigned int actionfps = 30;

 /*
@@ -124,7 +124,7 @@
  * 6: Bar ("|")
  * 7: Snowman ("☃")
  */
-static unsigned int cursorshape = 2;
+static unsigned int cursorshape = 4;

 /*
  * Default colour and shape of the mouse cursor
